mkt/tvplace/tests/test_serializers.py

```diff
@@ -0,0 +1,95 @@
+from django.contrib.auth.models import AnonymousUser
+from django.test.client import RequestFactory
+
+from nose.tools import ok_
+
+from mkt.site.tests import ESTestCase, TestCase, app_factory
+from mkt.tags.models import Tag
+from mkt.tvplace.serializers import (TVAppSerializer, TVESAppSerializer,
+                                     TVWebsiteSerializer,
+                                     TVESWebsiteSerializer)
+from mkt.webapps.indexers import WebappIndexer
+from mkt.websites.indexers import WebsiteIndexer
+from mkt.websites.utils import website_factory
+
+
+class AppSerializerTestsMixin(object):
+
+    def setUp(self):
```

Comment:
Ah, I wonder if that's going to bite us, particularly in the ES tests, which do need to call `ESTestCase.setUp()`... I'm starting to regret my mixin suggestion... Damn test classes, it's annoying to end up with something clean.